colortv/docs/androidTVSDK.md

```diff
@@ -792,19 +801,196 @@ Check our example [item layouts](https://github.com/color-tv/android-SampleApp/t
+* **Animation:** Hides when auto play timer is not used and when timer reaches 0, or toggles visibility when video is paused and resumed in case of using auto start functionality.
+* **Device:** ALL
+<br /><br />
+
+#### **ID:** ctv_videoPreviewContainer
+* **View type:** Any view extending the ViewGroup class (eg. `LinearLayout`, `FrameLayout`)
+* **Description:** Used to inject our `VideoPlayer` for previews playback.
+* **Animation:** Plays the preview and hides the thumbnail when video preview is enabled in UpNext interface (only if the preview is available).
+* **Device:** ALL
+<br /><br />
+
+#### **ID:** ctv_ivVideoThumbnail
+* **View type:** `ImageView`
+* **Description:** Used to display the video thumbnail.
+* **Animation:** If there is preview available and `ctv_videoPreviewContainer` is present in the layout it is hidden when the preview is enabled in UpNext interface.
+* **Device:** ALL
+<br /><br />
+
+#### **ID:** ctv_tvTitle
+* **View type:** `TextView`
+* **Description:** Displays the video title.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvDescription
+* **View type:** `TextView`
+* **Description:** Displays the video description.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvDuration
+* **View type:** `TextView`
+* **Description:** Displays the video duration.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvSeries
+* **View type:** `TextView`
+* **Description:** Displays series name when recommending series.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvSeason
+* **View type:** `TextView`
+* **Description:** Appends series season number to contained by the view text, while recommending series. I.e. when TextView has set in XML file a text "Season: " it will append season nb. creating "Season: 1".
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvEpisode
+* **View type:** `TextView`
+* **Description:** Appends series episode number to contained by the view text, while recommending series. I.e. when TextView has set in XML file a text "Episode: " it will append episode nb. creating "Episode: 1".
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_clickable
+* **View type:** Any view extending the View class (eg. `LinearLayout`, `Button`)
+* **Description:** Element used to capture click on recommendation, which invokes `onContentChosen` callback. If this element is absent `ctv_upNext` is such element. It should be used only when having other views in UpNext which can be focusable and clickable. For TVs it should use some selector in order to point out an user that focus has been changed.
```

Comment:
They can do whatever they want and actually I'm using in samples "state_focused"